docs/hoon/syntax.md

```diff
@@ -97,100 +94,114 @@ fas /           pat @               wut ?
 zap !
 ```
 
-You don't need to memorize these glyph names, but it helps.  Code
-is often vocalized or subvocalized.   "pal" is easier to say,
-aloud or silently, then "left paren."
+You don't need to memorize these glyph names, but it might help. 
+Code is often vocalized or subvocalized. "pal" is easier to say,
+aloud or silently, than "left paren".
 
 Note that the list includes two separate whitespace forms: `ace`
-for a single space, `gap` for 2+, comment or newline.  In Hoon,
+for a single space, `gap` for 2+, comment, and newline.  In Hoon,
 the only significance in whitespace is the difference between
-`ace` and `gap`.
+`ace` and `gap`. Comments start with `::` and run to the end of 
+the line.
+
+An 80-column right margin is strongly encouraged.  Really 
+well-groomed Hoon uses a 55-column code margin and puts a standard 
+line comment at column 57.
 
-Comments start with `::` and run to the end of the line.  Hard
-tab characters are illegal, and an 80-column right margin is
-strongly encouraged.  Really well-groomed Hoon uses a 55-column
-code margin and puts a standard line comment at column 57.
+## Expressions and Runes
 
-Hoon does not accept non-ASCII text in source files, except UTF-8
-in quoted strings.
+Each regular Hoon expression begins with a  
+[rune](https://urbit.org/docs/about/glossary#rune). A rune is a pair 
+of ASCII punctuation marks (a digraph)&mdash;e.g., `:-`.  The first 
+glyph in the rune indicates the category&mdash;e.g., `:` runes make 
+[cells](https://urbit.org/docs/about/glossary#cell) (i.e., ordered 
+pairs). We pronounce runes using their glyph names&mdash;for `:-`, 
+we say "colhep".
+
+Each rune is followed by one or more subexpressions. The number and 
+kind of subexpressions depend on the rune.
+
+```
+:-
+  25
+3
```

Comment:
I'm 100% opposed to bare runes on their own line. In this case, I have a strong preference for

```
:-  25
3
```

Reply:
I'm open to whatever seems good here.  I have no strong views about this.